database/migrations/2015_10_25_125123_add_talk_revision_slides.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddTalkRevisionSlides extends Migration
```

Comment:
Yep :+1: We need to add a `nitpick.json` to ignore the tests dir. I'll do that elsewhere